src/ui/SimpleFilter/SimpleFilter.ts

```diff
@@ -316,6 +319,11 @@ export class SimpleFilter extends Component {
         this.circleElement.addClass('coveo-simplefilter-circle-hidden');
       }
     }
+
+    if (selectedValues.length == 0) {
+      this.isSticky = false;
```

Comment:
Hmm, if there are no selected values, we do not want to "stick" what exactly?
Also, could this behaviour maybe be captured by a test?